.husky/pre-commit

```diff
@@ -1,8 +1,13 @@
 STAGED_FILES_TO_LINT=$(git diff --name-only --cached --diff-filter=d -- "*.ts" "*.js")
+STAGED_FILES_TO_ANALYZE=$(git diff --name-only --cached --diff-filter=d -- "{packages,tools}/*/src/**/!(*.css).ts")
```

Comment:
This is nice! :)

Reply:
What do you mean by "globbing through packages"? Git diff is the _only_ way, as far as I know, to fetch the list of files changed: https://git-scm.com/docs/git-diff